pkg/apis/kops/cluster.go

```diff
@@ -58,6 +58,8 @@ type ClusterSpec struct {
 	KubernetesVersion string `json:"kubernetesVersion,omitempty"`
 	// Configuration of subnets we are targeting
 	Subnets []ClusterSubnetSpec `json:"subnets,omitempty"`
+	// AdditionalRoutes is used to add more routes of precreated nat instance or VPC peering connection
+	AdditionalRoutes []AdditionalRoutesSpec `json:"additionalRoutes,omitempty"`
```

Comment:
Any ideas on a better name? I stink at names. 

We have routes for egress, where they are vpn or nat instances.

EgressRoutes? NatRoutes? Any ideas?

Reply:
I don't like the name too. But, I am trying to follow the api convention. For example, to configure more **precreated** security groups for instance groups you set "additionalSecurityGroups". I am fine with more better name. For "NatRoutes", I don't think that would fit, the feature allow nat instance route and also VPC peering route. EgressRoutes could be fine. I am not sure. What do you think?

Reply:
@justinsb @KashifSaadat @gambol99 @blakebarnett any ideas on a better name?